README.md

```diff
@@ -36,6 +45,33 @@ Finally, install the package with
 ```bash
 pip install migmose
 ```
+## Features and how to use them
+
+At this point, MIG_mose works as a command-line interface tool (CLI). There are several flags available to provide a user-friendly way to interact with MIG_mose.
+Below the available options are listed:
+
+- **Input Directory (`-i`, `--input_dir`):**
+    - Description: This option allows the user to specify the path to the directory containing the .docx files for the MIGs (Message Implementation Guides).
+    - Example: `--input_dir /path/to/input_directory`
+
+- **Message Format (`-mf`, `--message_format`):**
+    - Description: This option defines the set of message formats to be parsed. Users can specify multiple message formats by providing multiple values. The formats are EdiFact formats (cf. [maus.edifact.EdifactFormat](https://github.com/Hochfrequenz/mig_ahb_utility_stack/blob/main/src/maus/edifact.py)).
+    - Example: `--message_format "UTILMD" --message_format "ORDCHG"`
```

Comment:
gibt es auch eine Möglichkeit alle Formate zu triggern ohne dass ich alle einzeln aufzählen muss?

Reply:
hinzugefügt in 
0ad990671c4ada6b8de4825ec0c9dacab20885e1
und
b3b710e9dba2b2c6428ee9e14e79efea34668ed3